### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by ksin**
Initial layout for the home page. Most of the formatting is there. Follow-up tasks:

- [ ] General formatting and copy cleanup.
- [ ] Donate button needs to link to the donate page.
- [ ] Someone to provide alt text for images and video.
- [ ] Update video to not mention Code Cooperative.

Getting the image grid and spacing perfectly 1:1 with the designs is pretty tough. Maybe someone with better CSS can take a look to follow-up. 

homepage

### CODE REVIEW

File: src/_assets/css/abstracts/_tokens.scss [hunk 1]
```diff
@@ -30,6 +30,7 @@
   --sp-loose: 2rem;
   --sp-looser: 3rem;
   --sp-max: 6rem;
+  --ls-loose: 0.11em;
```
Comment (ksin) [inline src/_assets/css/abstracts/_tokens.scss:33]: letter-spacing

File: src/_assets/css/abstracts/_tokens.scss [hunk 2]
```diff
@@ -6,8 +6,8 @@
   --yellow-500: #fff42c;
   --white-500: #ffffff;
   --white-200: #f2efe4;
-  --gray-300: #f3f3f3;
-  --gray-400: #828282;
+  --gray-300: #BEBEBE;
```
Comment (ksin) [inline src/_assets/css/abstracts/_tokens.scss:9]: saw these values on figma instead so updated accordingly

File: src/components/WaterColor/WaterColor.js [hunk 3]
```diff
@@ -1,14 +1,14 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-import WebPackage from '_assets/images/sketch-webpage.png'
+import WebPage from '_assets/images/sketch-webpage.png'
```
Comment (ksin) [inline src/components/WaterColor/WaterColor.js:4]: fixed typo

File: src/pages/index.module.scss [hunk 4]
```diff
@@ -1,7 +1,36 @@
...
+  color: var(--gray-300);
+  letter-spacing: var(--ls-loose);
+  font-family: var(--font-mono);
+  line-height: var(--lh-loose);
+  position: absolute;
+  right: 0;
+  width: 30rem;
+}
+
+.gridContainer12 {
```
Comment (ksin) [inline src/pages/index.module.scss:18]: Can probably extract this grid stuff into a component, but don't want to do too much in one PR

**Comment by adekunleoduye** [comment]
Looks good. We will need to improve the mobile experience a bit.

image

image


We can chat with Savannah on how we should tackle.

**Comment by goizueta** [comment]
@adekunleoduye @ksin is this PR ready to be merged into rebrand once the conflicts are resolved?

**Comment by noidontdig** [comment]
Screen Shot 2020-07-10 at 9 30 13 AM

[ ] Light font weight should be 300 and bold font weight should be a little less bold (but I think Adekunle is making these changes for other pages)

[ ] I kind of want the video to autoplay when I land on the page or scroll down

Screen Shot 2020-07-10 at 9 54 00 AM

[ ] Font weight should be 300 here too for both font sizes. Using the right font weights across the site will fix this. 

Screen Shot 2020-07-10 at 9 54 07 AM

[ ] Brush stroke should be behind the illustration (change z-index)

**Comment by noidontdig** [comment]
COPY UPDATES

1. Header -- can we try this? 

Screen Shot 2020-07-10 at 10 26 46 AM


2. Under the video: _Emergent Works interrupts the prison cycle  by creating high-paying jobs in technology  for formerly incarcerated people._ 

Screen Shot 2020-07-10 at 11 56 47 AM

3. 
**We are a nonprofit technology company that trains and employs people coming home  from prison.**
Through our projects, our formerly incarcerated apprentices get valuable paid work experience to kickstart their careers in tech.

**Comment by ksin** [comment]
I'm just using existing tags and classes for the fonts, so I'll leave it as is, assuming Adekunle's changes will update things accordingly. Otherwise, we can go back in to fix them in a follow-up PR.

Copy has been updated.

In my experience, autoplaying a video on page load isn't a good experience, especially for mobile. Are you sure you want that?

Tackled the mobile UI as best I can.

There's one more Lorem Ipsum under "The Emergent Works community is open and endless." that doesn't have copy. Can you provide?

**Pull request merged**